src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IVsFileChangeExFactory.cs

```diff
@@ -1,8 +1,9 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.VisualStudio.Shell.Interop;
 using Moq;
 
-namespace Microsoft.VisualStudio.Shell.Interop
+namespace Microsoft.VisualStudio.ProjectSystem
```

Comment:
I see. I saw that all other mocks were under the ProjectSystem namespace and so moved it there but I suppose all the types I looked at were also in the ProjectSystem namespace. I'll move it back to be consistent.